pyproject.toml

```diff
@@ -11,7 +11,7 @@ python = "^3.10"
 httpx = "^0.24.0"
 locust = "^2.15.0"
 click = "^8.1.3"
-
+pandas = "^2.0.2"
```

Comment:
Do we really need to include pandas in dependencies? It's quite a heavy library to use it only for `to_timedelta`. I would recommend using a more lightweight solution.